Aptoide/appcoins-wallet-android
# [MOB-1044] Improved wallet validation state layout

**What does this PR do?**

Adds the changes made on zeplin on the wallet validation state

**Database changed?**

   No

**Where should the reviewer start?**

fragment_balance.xml

**How should this be manually tested?**

Check these 3 use cases:
Wallet verified
Wallet unverified
Wallet unverified without network connection

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/MOB-1044


**Code Review Checklist**
